src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this.result;
```

Comment:
```suggestion
      return this.result + number;
```
anyway you rewrite `this.result` in `operate`. fix for all methods